### PR TITLE
support ocfs2 log collecting

### PR DESCRIPTION
**Pull request by losemyheaven**
Hi Krig,

This pr intends to add support for collecting ocfs2 logs.

Sometimes, our customer need this to report log to us.

And I also need your suggestion.

Thanks.

### CODE REVIEW

File: hb_report/utillib.py [hunk 1]
```diff
@@ -1629,6 +1630,20 @@ def time_status():
     time_f = os.path.join(constants.WORKDIR, constants.TIME_F)
     crmutils.str2file(out_string, time_f)
 
+def dump_ocfs2():
+    ocfs2_f = os.path.join(constants.WORKDIR, constants.OCFS2_F)
+    f = open(ocfs2_f, "w")
```
Comment (krig) [inline hb_report/utillib.py]: Two recommendations:

1. Make sure that you check and handle the case if opening the file fails for whatever reason.

2. Use the Python `with` syntax to close the file:

```
with open(ocfs2_f, "w") as f:
   ...
```

File: hb_report/utillib.py [hunk 2]
```diff
@@ -1629,6 +1630,20 @@ def time_status():
...
+
+    #dump all tasks stack into dmesg
+    cmd = "echo t > /proc/sysrq-trigger"
+    crmutils.get_stdout(cmd)
```
Comment (krig) [inline hb_report/utillib.py]: Hmm, this looks odd to me. Maybe use `os.system("echo t > /proc/sysrq-trigger")` instead?

Comment (losemyheaven) [reply hb_report/utillib.py]: Thanks krig,
I'll do it.
: )

**Comment by losemyheaven** [comment]
Thanks krig,
I'll do it.
: )

**Pull request closed without merging**